### ISSUE TITLE
BUG:  The markdown embedded by Frame Tool can't be displayed correctly in full screen mode.

### ISSUE THREAD

**Issue by you-n-g** [zsviczian/obsidian-excalidraw-plugin#1197]
**Your environment**
Please run `Command Palette/Show Debug info` in Obsidian and paste the result here.
```
SYSTEM INFO:
	Obsidian version: v1.3.5
	Installer version: v1.2.7
	Operating system: Windows 10 Enterprise 10.0.22621
	Login status: not logged in
	Insider build toggle: off
	Live preview: on
	Legacy editor: off
	Base theme: light
	Community theme: none
	Snippets enabled: 0
	Restricted mode: off
	Plugins installed: 13
	Plugins enabled: 10
		1: Advanced Slides v1.20.0
		2: Remotely Save v0.3.25
		3: Text Generator v0.3.20
		4: Icon Folder v2.1.2
		5: Various Complements v8.3.1
		6: Settings Search v1.3.7
		7: Paste URL into selection v1.7.0
		8: Diagrams v1.5.0
		9: Emoji Toolbar v0.4.1
		10: Excalidraw v1.9.7

RECOMMENDATIONS:
	Community plugins: for bugs, please first try updating all your plugins to latest. If still not fixed, please try to make the issue happen in the Sandbox Vault or disable community plugins.
```

**Describe the bug**
The markdown embedded by Frame Tool can't be displayed correctly in full screen mode.
![image](https://github.com/zsviczian/obsidian-excalidraw-plugin/assets/465606/ca2939f1-25dc-48ca-9a57-fcfb0c49fca0)

**To Reproduce**
1. Create a excalidraw file.
2. Insert a markdown file via `Insert File From Vault`
3. `Excalidraw: Toggle fullscreem model`

**Expected behavior**
The mar

**Screenshots**
![image](https://github.com/zsviczian/obsidian-excalidraw-plugin/assets/465606/ca2939f1-25dc-48ca-9a57-fcfb0c49fca0)

**Additional context**

**Comment by zsviczian** [comment]
Good catch. I did not try this feature in fullscreen mode... and indeed it does not work. Will look into it.

**Comment by zsviczian** [comment]
I fixed it in 1.9.8. Will release the update in a couple of days. I want to include some other fixes as well.

**Comment by you-n-g** [comment]
Amazing!!!
